cms/static_src/javascript/components/details.js

```diff
@@ -0,0 +1,17 @@
+class detailsToggle {
+  static selector() {
+    return 'details';
+  }
+
+  constructor(node) {
+    // get all <details> elements
+    const allDetails = document.querySelectorAll('details');
+
+    // collapse all <details> by default
+    allDetails.forEach((detail) => {
+        detail.removeAttribute('open');
+    });
```

Comment:
The `initComponent` function already runs `querySelectorAll` using the selector you define above, and then runs the constructor on each node found:

https://github.com/ONSdigital/dis-wagtail/blob/981bd844d505b44be15e64d2889b2ec7212422e7/cms/static_src/javascript/main.js#L5-L8

So all you really need to to is call `removeAttribute('open')` on the node:

```suggestion
    node.removeAttribute('open');
```